### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by fluteds**
**Corrected the branch mistakes from #1767** 

### Updated the readme for Dogecoin! 

This can be used for consideration as it felt really outdated and less doge.

### Changes: 

- Removed dead image links.
- Updated Travis ci badge link to direct to the master branch and the new Travis website url.
- Added Emojis to headers for clarity and to get the feel of dogecoin for newcomers. 
- Bolded some sub headers for clarity. e.g. block reward sections.
- Code formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,44 +104,44 @@ The original block reward schedule, with one-minute block targets and four-hour
 
   - [OSX Build Notes](doc/build-osx.md)
   - [Unix Build Notes](doc/build-unix.md)
-  - [Windows Build Notes](doc/build-windows.md)
+  - [Windows Build Notes](doc/build-msw.md)
```
Comment (patricklodder) [inline README.md:107]: this reverts another commit, needs to go out

File: README.md [hunk 2]
```diff
@@ -96,44 +104,44 @@ The original block reward schedule, with one-minute block targets and four-hour
...
 **DEBUG_LOCKORDER**
 
 Dogecoin Core is a multithreaded application, and deadlocks or other multithreading bugs
-can be very difficult to track down. Compiling with -DDEBUG_LOCKORDER (configure
-CXXFLAGS="-DDEBUG_LOCKORDER -g") inserts run-time checks to keep track of which locks
+can be very difficult to track down. Compiling with `-DDEBUG_LOCKORDER` (`configure
+CXXFLAGS="-DDEBUG_LOCKORDER -g"`) inserts run-time checks to keep track of which locks
 are held, and adds warnings to the debug.log file if inconsistencies are detected.
+
+#
```
Comment (patricklodder) [inline README.md:147]: this needs to go

File: README.md [hunk 3]
```diff
@@ -96,44 +104,44 @@ The original block reward schedule, with one-minute block targets and four-hour
...
+- P2P 22556
 
-![](https://dogesay.com/3f5a00acf72df93528b6bb7cd0a4fd0c.jpeg)
+## Development tips and tricks 🖥️📝
 
-## Development tips and tricks
+**Compiling for debugging**
 
-**compiling for debugging**
+Run `configure` with the `--enable-debug` option, then `make`. Or run configure with
```
Comment (patricklodder) [inline README.md:117]: please rebase your commit off #1766 / https://github.com/leuqarte/dogecoin/tree/1.14.3-dev to not have the same changes twice.

File: README.md [hunk 4]
```diff
@@ -96,44 +104,44 @@ The original block reward schedule, with one-minute block targets and four-hour
...
+- RPC 22555
+- P2P 22556
 
-![](https://dogesay.com/3f5a00acf72df93528b6bb7cd0a4fd0c.jpeg)
+## Development tips and tricks 🖥️📝
 
-## Development tips and tricks
+**Compiling for debugging**
 
-**compiling for debugging**
```
Comment (patricklodder) [inline README.md:109]: Not sure what changed here?

File: README.md [hunk 5]
```diff
@@ -1,23 +1,31 @@
...
+[![Build Status](https://travis-ci.com/dogecoin/dogecoin.svg?branch=master)](https://travis-ci.com/dogecoin/dogecoin) [![tip for next commit](https://tip4commit.com/projects/702.svg)](https://tip4commit.com/github/dogecoin/dogecoin)
 
-[![Build Status](https://travis-ci.org/dogecoin/dogecoin.svg?branch=1.14-dev)](https://travis-ci.org/dogecoin/dogecoin) [![tip for next commit](https://tip4commit.com/projects/702.svg)](https://tip4commit.com/github/dogecoin/dogecoin)
+## What is Dogecoin? – Such coin 🤔  
 
-## What is Dogecoin? – Such coin
-Dogecoin is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
+Dogecoin (styled as DOGE or Ð) is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
+- **Website:** [dogecoin.com.](https://dogecoin.com)
+- **Get started with doge here:** [dogecoin.com/getting-started.](https://dogecoin.com/getting-started.)
```
Comment (patricklodder) [inline README.md:12]: That section is outdated right now, the main page actually has better instructions.

File: README.md [hunk 6]
```diff
@@ -1,23 +1,31 @@
...
 
-https://dogecoin.com/
+## Wallets – Wow much hodl 🐕‍🦺
+### **Official Dogecoin Wallet** <img src="http://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png" width="25" height="25">
+- Supports Windows, Linux, MacOS and Android!
+- Get your paws on the [**official download!**](dogecoin.com/getting-started)
```
Comment (patricklodder) [inline README.md:17]: Not sure what this adds...

File: README.md [hunk 7]
```diff
@@ -1,23 +1,31 @@
-# Dogecoin Core [DOGE, Ð]
-==========================
+<p align="center">
+<img src="http://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png"
+</p>
+<h1 align="center">Dogecoin Core [DOGE, Ð]</h1>
```
Comment (patricklodder) [inline README.md:4]: Better to keep the markdown, as that's readable in text editors.

**Review by patricklodder — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (fluteds) [reply README.md:109]: Capitalised 'compiling'

File: README.md (hunk 5 above)
Comment (fluteds) [reply README.md:12]: Do you want me to use the section from the Master branch?

File: README.md (hunk 6 above)
Comment (fluteds) [reply README.md:17]: There felt like there was no information of wallets for doge... I can remove if you feel it doesn't fit the flow or adds anything.

File: README.md (hunk 7 above)
Comment (fluteds) [reply README.md:4]: Noted. Will revert back to markdown in next commit.

File: README.md [hunk 8]
```diff
@@ -45,18 +53,18 @@ code. This is especially important for large or high-risk changes. It is useful
...
 
-### How much doge can exist? – So many puppies!
+### How much doge can exist? – So many puppies! 🐕
 Early 2015 (approximately a year and a half after release) there will be approximately 100,000,000,000 coins.
 Each subsequent block will grant 10,000 coins to encourage miners to continue to secure the network and make up for lost wallets on hard drives/phones/lost encryption passwords/etc.
 
-### How to get doge? – To the moon!
-Dogecoin uses a simplified variant of the scrypt key derivation function as its proof of work with a target time of one minute per block and difficulty readjustment after every block. The block rewards are fixed and halve every 100,000 blocks. Starting with the 600,000th block, a permanent reward of 10,000 Dogecoin per block will be issued. 
+### How to get doge? – To the moon! 🚀🌕
+Dogecoin uses a simplified variant of the scrypt key derivation function as its proof of work with a target time of one minute per block and difficulty readjustment after every block. The block rewards are fixed and halve every 100,000 blocks. Starting with the 600,000th block, a permanent reward of 10,000 Dogecoin per block will be paid.
```
[the remark this answers is not in the document]
Comment (fluteds) [reply README.md:55]: No rationale just error on editing the readme from master branch.

File: README.md (hunk 6 above)
Comment (fluteds) [reply README.md]: Removed wallet from the readme.

File: README.md (hunk 5 above)
Comment (fluteds) [reply README.md]: Removed the getting started in favour of the main dogecoin.com website.

**Comment by patricklodder** [comment]
We've merged the preceding PR, could you please rebase off 1.14.3-dev instead now and resolve the merge conflict?

Thanks!

**Review by langerhans — DISMISSED** [review]
Needs rebase

**Review by patricklodder — APPROVED** [review]

**Review by rnicoll — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -1,18 +1,18 @@
...
+- **Website:** [dogecoin.com.](https://dogecoin.com)
 
-## License – Much license
+## License – Much license ⚖️
 Dogecoin Core is released under the terms of the MIT license. See [COPYING](COPYING) for more
-information or see https://opensource.org/licenses/MIT.
+information or see [opensource.org/licenses/MIT.](https://opensource.org/licenses/MIT)
+=======
+## What is Dogecoin? – Such coin
+Dogecoin is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
```
Comment (rnicoll) [inline README.md:15]:
```suggestion
=======
## What is Dogecoin? – Such coin
Dogecoin is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
```

**Review by rnicoll — CHANGES_REQUESTED** [review]
There's one bit of duplicate content from the merge, then can re-approve

File: README.md [hunk 10]
```diff
@@ -102,38 +102,34 @@ The original block reward schedule, with one-minute block targets and four-hour
 RPC 22555
 P2P 22556
 
-![](https://dogesay.com/3f5a00acf72df93528b6bb7cd0a4fd0c.jpeg)
+## Development tips and tricks 💻📝
 
-## Development tips and tricks
+Run `configure` with the `--enable-debug` option, then `make`. Or run configure with `CXXFLAGS="-g -ggdb -O0"` or whatever debug flags you need.
```
Comment (patricklodder) [inline README.md:107]: another duplicate

**Comment by patricklodder** [comment]
I have cherry-picked your work and sanitized it manually in #1774 to not make this block release. Have used your original commit as a baseline and retained attribution to you.

Thank you for this contribution, we appreciate it! I will close this PR in favor of the sanitized one.

**Pull request closed without merging**